### PR TITLE
Add vision (image description)

### PR DESCRIPTION
**Pull request by chadbailey59**

### CODE REVIEW

File: src/dailyai/services/daily_transport_service.py [hunk 1]
```diff
@@ -225,6 +229,16 @@ def _post_run(self):
         self.client.leave()
         self.client.release()
 
+    def _handle_video_frame(self, participant_id, video_frame):
+        # TODO-CB: What about multiple participants?
```
Comment (aconchillo) [inline src/dailyai/services/daily_transport_service.py:233]: Just saw this comment... I think you would need a `receive_queue` per participant.

**Comment by chadbailey59** [comment]
...aaand we've got our first `autopep8` fight. I can see the changes that the linter wants, but even when I explicitly "format document" with autopep8 in VS code, it doesn't make those changes. Is there a secret autopep8 config file somewhere?

**Comment by philtsip** [comment]
My 2 suggestions:

1. remove this line: from dailyai.services.deepgram_ai_services import DeepgramTTSService since you aren't using Deepgram TTS in this example

2. Either use Speech-to-text explicitly and request the right API key for it, or set start_transcription=False. Otherwise, a user gets this cryptic error: 40 2024-03-19 01:19:49,832 Transcription error: Either set "deepgram:DEEPGRAM_API_KEY" at domain/room config or contact support to enable transcription for your domain

**Comment by aconchillo** [comment]
> ...aaand we've got our first `autopep8` fight. I can see the changes that the linter wants, but even when I explicitly "format document" with autopep8 in VS code, it doesn't make those changes. Is there a secret autopep8 config file somewhere?

Sorry @chadbailey59. I'll see if I cna figure it out with VSCode

**Comment by aconchillo** [comment]
@chadbailey59 Configure your VSCode autopep8 extendsion with these arguments: 

image

Also, enable formatting on save (check the extension documentation for how to do it).

**Comment by aconchillo** [comment]
I'll update the hacking section of the README with these VS Code requirements.

**Comment by aconchillo** [comment]
> I'll update the hacking section of the README with these VS Code requirements.

See https://github.com/daily-co/daily-ai-sdk/pull/78

**Comment by aconchillo** [comment]
This PR is now obsolete. Closing.

**Pull request closed without merging**